project/ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,109 @@
+## Triaging of Issues
+
+Docker has a lot of issues and it can be sometimes daunting to look through
+everything to find something to work on.  Triaging issues is important to
+help both the author convey the real problem or request that they have and
+to help developers easily find something to work on.
+
+Remember there are more ways than just code that you can contribute to an
+open source project and the community would really thank you if you spent
+spend time triaging issues.
+
+
+### Important first steps
+
+Before jumping into an issue very far make sure that the author of the issue
+has provide enough information so that you are able to make an educated 
+reccomendation on how this is to be triaged.  Standard information that *must*
```

Comment:
Shouldn't we accept that an issue lacks these informations as long as we can reproduce on a given setup and provide the missing information ourselves? I'm afraid we'd be closing way too many.